### ISSUE TITLE
Dataset Open

### ISSUE THREAD

**Issue by Lijiaxin0111** [JarrentWu1031/AnimateAnyMesh#3]
Wonderful Work!
When will your dataset Dymesh open-source?

**Comment by JarrentWu1031** [comment]
> Wonderful Work! When will your dataset Dymesh open-source?

Thanks for your interest! We are planning to release DyMesh dataset in August, along with the code.

**Comment by zyhbili** [comment]
> > Amazing Work! Would you release your data curation code? I am a new comer in this area. I encounter difficulties when dealing with GLB format data in Diffusion4D. Do you first extract and save per-frame mesh using Blender? Could the dynamic mesh sequence be directly extracted from GLB files using only python scripts?
> 
> Thank you for the appreciation! We used Blender's Python extension 'bpy' to complete the vertex trajectory extraction for all GLB files. The extracted files mainly consist of T_N_3 vertex trajectories and M*3 faces sequences. This process can be fully completed using bpy, and we will open-source the corresponding extraction scripts along with the processed data.

Thanks for your quick reply!  Besides, i wonder the data normalization details. For example, do you re-center and re-scale the mesh sequences into the canonical volume space i.e range [0,1] ?

**Comment by zyhbili** [comment]
Hi!  Thank you for releasing the model recently. When will the processed dataset and the extraction scripts be released?

**Comment by zyhbili** [comment]
@JarrentWu1031 Thanks for your sharing! I am downloading the 16f dataset. Could you update the dataloader file?

**Comment by JarrentWu1031** [comment]
> [@JarrentWu1031](https://github.com/JarrentWu1031) Thanks for your sharing! I am downloading the 16f dataset. Could you update the dataloader file, including the train, val, test spilt.

Fine! We will release the val/test-set soon! The val/test-set are absent from the data in [DyMesh_16](fhttps://www.modelscope.cn/datasets/jarrentwu/DyMesh_16f). Please use the currently publicly available data as the training-set.

**Comment by JarrentWu1031** [comment]
> [@JarrentWu1031](https://github.com/JarrentWu1031) Thanks for your sharing! I am downloading the 16f dataset. Could you update the dataloader file, including the train, val, test spilt.

Fine! We will release the val/test-set soon! The val/test-set are absent from the data in [DyMesh_16f](https://www.modelscope.cn/datasets/jarrentwu/DyMesh_16f). Please use the currently publicly available data as the training-set.

**Comment by zyhbili** [comment]
Thanks for your reply, i am trying to finetune the DyMeshVAE. Would you release the dataloader files? I want to check my data preprocessing details.

**Comment by JarrentWu1031** [comment]
You're welcome! Please refer to test_dvae.py for the data process pipeline, it is basically the same with the training script. The only thing you need to do is to rewrite it as a dataloader.
